Package.swift

```diff
@@ -13,7 +13,7 @@ let package = Package(
     ],
     targets: [
         .binaryTarget(name: "TaboolaSDK",
-                      url: "https://taboola-mobile-sdk.s3-us-west-2.amazonaws.com/ios/sdk-binaries/xcframework/official/4.0.2/TaboolaSDK.xcframework.zip",
-                      checksum: "4c9acc439d38a04734a855891dd5e55a296f4d29be04a916bb26504a52d6a5f4")
+                      url: "https://taboola-mobile-sdk.s3.us-west-2.amazonaws.com/ios/sdk-binaries/xcframework/beta/4.0.3/meta-1/TaboolaSDK.xcframework.zip",
```

Comment:
did you checked this URL? 
There is a different in `s3-us-west-2` into `s3.us-west-2`